### PR TITLE
Update MDs and release publishing files to reflect the repo move to OWF

### PR DESCRIPTION
**Pull request by swcurran**
A complementary PR to the one that @jamshale made - #3267. His moves the technical components -- code and GHAs -- while this one focuses on documentation and the release/publishing process. I don't think there is a lot of overlap between them -- hopefully none.

Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

**Review by WadeBarnes — COMMENTED** [review]
Would it be possible to use relative links to the various files rather than an explicit URL?  Since you're having to update all the references anyway, it may be a good time to do this.

**Comment by swcurran** [comment]
It is not always straight forward. The links have to work in both the GitHub repo context and on https://aca-py.org, so there are places where I just use the hardcoded link (as you have seen), some where there is an update made to the link in generating the gh-pages site, and others that “just work”.   Things that ONLY exist in the GitHub repo MUST be absolute links for them to work on the gh-pages site.   I did clean up a bunch of them in this PR, but not all can be done.

So, bottom line, no.

File: sonar-project.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
-sonar.projectKey=hyperledger_aries-cloudagent-python
-sonar.organization=hyperledger
-sonar.projectName=aries-cloudagent-python
+sonar.projectKey=penwallet-foundation_acapy
```
Comment (jamshale) [inline sonar-project.properties:1]: I'd revert this change. It's covered in my PR. If you do include it please fix the spelling mistake.

Comment (swcurran) [reply sonar-project.properties]: Doh!! Should have known better.  Reverted.

**Review by jamshale — APPROVED** [review]
